tools/metadata/validate.py

```diff
@@ -51,7 +51,7 @@ def validate_content(textproto_content,
     log.warning('"url" field is deprecated, please use "identifier" instead')
 
   git_id = next((id for id in third_party.identifier if id.type == 'Git'), None)
-  if git_id:
+  if git_id and not os.path.exists('internal'):
```

Comment:
Can you add a comment explaining to rationale?

Reply:
done